versions/2016-10-31.md

```diff
@@ -1240,3 +1241,33 @@ Domain:
     EvaluateTargetHealth: Boolean # optional | default value is false
     DistributionDomainName: String # OPTIONAL if the EndpointConfiguration is EDGE | Default points to Api Gateway Distribution | Domain name of a [cloudfront distribution](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudfront-distribution.html)
 ```
+
+### Referable properties of SAM resources
+- [AWS::Serverless::Function](#referable-properties-of-serverless-function)
+- [AWS::Serverless::Api](#referable-properties-of-serverless-RestApi)
+- [AWS::Serverless::HttpApi](#referable-properties-of-serverless-HttpApi)
+
+#### Referable properties of Serverless Function
+Property Name | Reference | LogicalId | Description
+---|:---:|---|---
+Alias | `function-logical-id`.Alias | `function-logical-id`Alias`alias-name` | SAM generates an `AWS::Lambda::Alias` resource when `AutoPublishAlias` property is set. This resource can be referenced in intrinsic functions by using the resource logical ID or  `function-logical-id`.Alias
+Version | `function-logical-id`.Version | `function-logical-id`Version`sha` | SAM generates an `AWS::Lambda::Version` resource when `AutoPublishAlias` property is set. This resource can be referenced in intrinsic functions by using the resource logical ID or  `function-logical-id`.Version
+Destination (for SNS Topic) | `function-logical-id`.DestinationTopic |`function-logical-id`EventInvokeConfig`OnSuccess/OnFailure`Topic| SAM auto creates an `AWS::SNS::Topic` resource when `Destination` property of `DestinationConfig` property in `EventInvokeConfig` property is not specified. This generated resource can be referenced by using `function-logical-id`.DestinationTopic
+Destination (for SQS Queue) | `function-logical-id`.DestinationQueue |`function-logical-id`EventInvokeConfig`OnSuccess/OnFailure`Queue | SAM auto creates an `AWS::SQS::Queue` resource when `Destination` property of `DestinationConfig` property in `EventInvokeConfig` property is not specified. This generated resource can be referenced by using `function-logical-id`.DestinationQueue
+
+#### Referable properties of Serverless RestApi
+
+Property Name | Reference | LogicalId | Description
+---|:---:|---|---
+Stage | `restapi-logical-id`.Stage | `restapi-logical-id`Stage | SAM generates `AWS::ApiGateway::Stage` resource when `AWS::Serverless::Api` resource is defined. This resource can be referenced in intrinsic function using the resource logical id or `restapi-logical-id`.Stage
```

Comment:
Same thing here: `<api-logical-id><stage-name>Stage`

Reply:
thank you. I have updated it